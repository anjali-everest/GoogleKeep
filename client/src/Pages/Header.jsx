import { KEEP } from "../AppConstants";
import keepIcon from "../styles/keepIcon.png";
import React from "react";

const Header = () => {
    return (
        <nav className="header">
            <img src={keepIcon} alt='Keep-Logo' className="header-icon" />
            <h2 className="header-title">{KEEP}</h2>
            <br />
        </nav>
    );
};

export default Header;