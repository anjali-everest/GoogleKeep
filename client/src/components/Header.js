import { KEEP } from "../constants/appConstants";
import keepIcon from "./../assets/keepIcon.png";
import React from "react";

const Header = () => {
  return (
    <nav className="header">
      <img src={keepIcon} alt="Keep-Logo" className="header-icon" />
      <h2 className="header-title">{KEEP}</h2>
      <br />
    </nav>
  );
};

export default Header;
