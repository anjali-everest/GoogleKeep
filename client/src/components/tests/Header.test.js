import React from "react";
import { shallow } from "enzyme";
import Header from "../Header";

describe("Header tests", () => {
  it("Should render without crashing", () => {
    shallow(<Header />);
  });
  it("should match the Header snapshot", function () {
    const header = shallow(<Header />);
    expect(header).toMatchSnapshot();
  });
});
