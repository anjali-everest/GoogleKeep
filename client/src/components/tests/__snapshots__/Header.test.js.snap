// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header tests should match the Header snapshot 1`] = `ShallowWrapper {}`;
